app/sections/view-capabilities/macro.test.js

```diff
@@ -0,0 +1,120 @@
+        expect($('h3').text().trim()).toEqual('Capabilities met');
+        done();
+      });
+  });
+
+  it('should not render the solution capabilities section when not provided', (done) => {
+    const context = {
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        expect($('[data-test-id="view-solution-capabilities"]').length).toEqual(0);
+        done();
+      });
+  });
+
+  it('should render the capability expandable sections', (done) => {
+    const context = {
+      params: [
+        {
+          name: 'Prescribing',
+          version: '1.0',
+          description: 'Supports the effective and safe prescribing of medical products and appliances to Patients. Information to support prescribing will be available.',
+          link: 'http://www.some-prescribing-link.com',
+        },
+        {
+          name: 'Social Prescribing',
+          version: '1.0',
+          description: 'Supports the effective and safe prescribing of medical products and appliances to Patients. Information to support prescribing will be available.',
+          link: 'http://www.some-prescribing-link.com',
+        },
+      ],
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const viewCapabilities = $('[data-test-id="view-capabilities"]');
+        const viewSectionCapabilities = viewCapabilities.find('[data-test-id="view-section-capabilities"]');
+
+        expect(viewSectionCapabilities.length).toEqual(2);
+
+        done();
+      });
+  });
+
+  it('should render the capability expandable sections', (done) => {
+    const context = {
+      params: [
+        {
+          name: 'Prescribing',
+          version: '1.0',
+          description: 'Supports the effective and safe prescribing of medical products and appliances to Patients. Information to support prescribing will be available.',
+          link: 'http://www.some-prescribing-link.com',
+        },
+        {
+          name: 'Social Prescribing',
+          version: '1.0',
+          description: 'Supports the effective and safe prescribing of medical products and appliances to Patients. Information to support prescribing will be available.',
+          link: 'http://www.some-prescribing-link.com',
+        },
+      ],
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const viewCapabilities = $('[data-test-id="view-capabilities"]');
+        const viewSectionCapabilities = viewCapabilities.find('[data-test-id="view-section-capabilities"]');
+
+        expect(viewSectionCapabilities.length).toEqual(2);
+
+        done();
+      });
+  });
+
```

Comment:
I would add the following tests here too
the name of the expandable section (name + version) is rendered
the description is rendered
the link is rendered
The epic is rendered